### PR TITLE
add uyghur language pack

### PR DESCRIPTION
**Pull request by rustam255786**
add uyghur language pack

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Documentation
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [x] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] It's submitted to the `dev` branch and _not_ the `master` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] It's been tested on a Cordova (iOS, Android) app
- [ ] It's been tested on a Electron app
- [ ] Any necessary documentation has been added or updated [in the docs](https://github.com/quasarframework/quasar/tree/dev/docs) (for faster update click on "Suggest an edit on GitHub" at bottom of page) or explained in the PR's description.

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

**Comment by Ansar-786** [comment]
added Uyghur language pack

File: ui/lang/ug.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  date: {
+    days: days,
+    daysShort: 'ي_د_س_چ_پ_ج_ش'.split('_'),
+    months: monthsShort,
+    monthsShort: monthsShort,
+    headerTitle: function(date, model) {
+      return monthsShort[model.month - 1] + 'نىڭ ' + model.day + '-كۈنى، ' + days[date.getDay()]
+    },
+    firstDayOfWeek: 1, // 0-6, 0 - Sunday, 1 Monday, ...
+    format24h: false
```
Comment (rstoenescu) [inline ui/lang/ug.js:32]: `pluralDay` missing

**Comment by rstoenescu** [comment]
Hi!

A very welcomed contribution.
My only comment is that date > pluralDay is missing. Can you add it, please?

**Comment by Ansar-786** [comment]
> 
> 
> Hi!
> 
> A very welcomed contribution.
> My only comment is that date > pluralDay is missing. Can you add it, please?

I am sorry, i don't pay attention this message immediately, i will added this missing

**Comment by rstoenescu** [comment]
Thanks! This will go into Quasar v1.14.1.

**Pull request merged**